.vscode/launch.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Has this file been committed by mistake? Should it be added to `gitignore` in general?